### ISSUE TITLE
Debian package

### ISSUE THREAD

**Issue by jrocha** [troglobit/mcjoin#24]
Hi there,

Would be possible to make mcjoin an official Debian package?
Looks like it is already possible to create a package, what would be needed to make it available over apt?

**Comment by troglobit** [comment]
Hi, we upstream maintainers have very little say so in packaging in distributions downstream from us. I've pre-packaged to ease adoption for an official debian maintainer, if they find it scratches an itch for them.  I've tried mysefl to become part of Debian, so I've seen the packaging I've done as practice, but I'm just a beginner.

I suggest you bring it up with Debian.

My own repo is available at https://deb.troglobit.com/ but only supports x86_64 and is built on a Debian "old stable", so the packages may not even start on a modern debian/ubuntu/mint.  If you're still interested, please see the README for details.

**Comment by rustyx** [comment]
Joachim, your tools are very, very useful. I think the Debian community would really benefit from having them onboard.
If you don't want to be a maintainer, that's fine, but at least [submit an RFP](https://www.debian.org/doc/manuals/developers-reference/pkgs.html#newpackage) to at least announce it to Debian.

**Comment by troglobit** [comment]
@rustyx thank you for the nice words, great to hear my tools are useful to other people! Let me rephrase my statement from above; the issue for me is time. I don't have the time to engage with all communities to promote my work, even if I'd very much like to as in the case of Debian.  I have LOT of stuff on my plate and many other priorities, and as I understand it, anyone can file an RFP not just the upstream author.  So you are very welcome to do so yourself.  

Cheers
 /Joachim
